liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/PackageServerMojo.java

```diff
@@ -35,10 +38,46 @@
 @Mojo(name = "package", defaultPhase = LifecyclePhase.PACKAGE)
 public class PackageServerMojo extends StartDebugMojoSupport {
 
+    public enum PackageFileType {
+        JAR("jar",".jar"),
+        TAR("tar",".tar"),
+        TARGZ("tar.gz",".tar.gz"),
+        ZIP("zip",".zip");
+
+        private final String value;
+        private final String fileExtension;
+
+        private PackageFileType(final String val, final String ext) {
+            this.value = val;
+            this.fileExtension = ext;
```

Comment:
Since the extension is just the value prepended with `.`, I don't think they need to be specified separately. Could just do `this.fileExtension = "." + this.value`.

Reply:
why? No need to create that String every time.

Reply:
See comment below about removing the extension and just keeping the value.